extension/src/components/wallet_details.rs

```diff
@@ -37,18 +39,46 @@ impl Component for WalletDetails {
 
     fn view(&self) -> Html {
         let Props { address, balances } = &self.props;
-
+        let address_svg_string = QrCode::with_error_correction_level(address, qrcode::EcLevel::H)
+            .map(|code| {
+                code.render::<svg::Color>()
+                    .max_dimensions(128, 128)
+                    .min_dimensions(128, 128)
+                    .build()
+            })
+            .unwrap();
+        let address_svg = string_to_svg(&address_svg_string);
+        // we need to provide a callback function to the input field. This one does nothing.
+        let update = Callback::default();
```

Comment:
It seems like the address should not be a text area since we don't want to be able to modify it. What about https://docs.rs/ybc/0.1.6/ybc/struct.Content.html or https://docs.rs/ybc/0.1.6/ybc/struct.Box.html?

Reply:
Or is this to be able to copy the address?

Reply:
Yea... I'm misusing textarea because otherwise it wouldn't allow us to select and copy...